pkgs/top-level/python-packages.nix

```diff
@@ -56,7 +56,9 @@ let
   # See build-setupcfg/default.nix for documentation.
   buildSetupcfg = import ../build-support/build-setupcfg self;
 
-  fetchPypi = callPackage ../development/interpreters/python/fetchpypi.nix { };
+  # fetchPypi -> fetchFromPyPI
+  fetchPypi = fetchFromPyPI;
```

Comment:
maybe we can deprecate this and remove in the (long) future?

Reply:
there's not a good way to deprecate within nixpkgs. If you emit warnings then that breaks a lot of CI's (including ofborg)

Reply:
> If you emit warnings then that breaks a lot of CI's (including ofborg)

Which is kinda stupid.

----

I see myself already doing this quite often so why not.

```suggestion
  fetchPypi = fetchFromPyPI;
  fetchFromPyPi = throw "Did you mean fetchFromPyPI?"
```

Reply:
Is this useful? The compiler will catch it anyway...

Reply:
I would say so, it's pretty common to do a typo like this, but it reads as correct.

Reply:
> Is this useful? The compiler will catch it anyway...

I am all about this alias. Same as `fetchFromGithub`.

Reply:
It looks like a case for a whole `mistaken-aliases.nix` file.